packages/kbn-es/src/artifact.js

```diff
@@ -194,11 +194,19 @@ exports.Artifact = class Artifact {
    * @param {string} dest
    * @return {Promise<void>}
    */
-  async download(dest) {
+  async download(dest, { useCached = false }) {
     await retry(this._log, async () => {
       const cacheMeta = cache.readMeta(dest);
       const tmpPath = `${dest}.tmp`;
 
+      if (useCached && cacheMeta.exists) {
```

Comment:
Do you think we should log a warning or error out if the cache doesn't exist?

Reply:
sounds like a good idea, in such case it will continue to download as normal so we can say something like "no cached snapshot found, use-cached has no effect" or something alike? wdyt?

Reply:
Sounds good, either option works for me.

Reply:
or do you mean throw an error and kill the script?

Reply:
I think either option would be reasonable, I'd probably kill the script if it was me but I'm not opposed to just logging a warning

Reply:
I can think of people using this option as something they will arrow-up on their terminal. killing the script will break that flow, and this case only exists if the user uses the option on the first run of an specific version.